website/docs/docs/core/connect-data-platform/fabric-setup.md

```diff
@@ -362,7 +362,7 @@ Valid values for `authentication`:
 * `ActiveDirectoryPassword`: Active Directory authentication using username and password
 * `ActiveDirectoryInteractive`: Active Directory authentication using a username and MFA prompts
 * `ActiveDirectoryIntegrated`: Active Directory authentication using the current user's credentials
```

Comment:
left "Active Directory" as is. happy to update these too if appropriate to do so